partner_event/models/event_event.py

```diff
@@ -2,6 +2,7 @@
 # Copyright 2015 Tecnativa S.L. - Javier Iniesta
 # Copyright 2016 Tecnativa S.L. - Antonio Espinosa
 # Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2020 Tecnativa S.L. - Víctor Martínez
```

Comment:
If no relevant changes in the logic of the code are done, you shouldn't add your self to author lines (only in contributors on the README.rst)

Reply:
Sorry, maybe some change was made initially and not in the end and for that reason it would be added.
It is already removed.